### PR TITLE
fix: Fix the issue of black background appearing in SearchEdit in sma…

### PR DESCRIPTION
**Pull request by robertkill**
…ll window mode under the white theme

Fix the issue of black background appearing in SearchEdit in small window mode under the white theme

Log:
Influence: window mode

### CODE REVIEW

**Review by 18202781743 — APPROVED** [review]

**Comment by xzl01** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/linuxdeepin/dde-launchpad/pull/104#pullrequestreview-1926992379" title="Approved">18202781743</a>*, *<a href="https://github.com/linuxdeepin/dde-launchpad/pull/104#" title="Author self-approved">robertkill</a>*


The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=linuxdeepin%2Fdde-launchpad).

<details open>
Needs approval from an approver in each of these files:

- **[debian/deepin/OWNERS](https://github.com/linuxdeepin/dde-launchpad/blob/master/debian/deepin/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by BLumia — APPROVED** [review]

**Comment by xzl01** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/linuxdeepin/dde-launchpad/pull/104#pullrequestreview-1926992379" title="Approved">18202781743</a>*, *<a href="https://github.com/linuxdeepin/dde-launchpad/pull/104#pullrequestreview-1927008982" title="Approved">BLumia</a>*, *<a href="https://github.com/linuxdeepin/dde-launchpad/pull/104#" title="Author self-approved">robertkill</a>*


The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=linuxdeepin%2Fdde-launchpad).

<details open>
Needs approval from an approver in each of these files:

- **[debian/deepin/OWNERS](https://github.com/linuxdeepin/dde-launchpad/blob/master/debian/deepin/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
